From roel.kluin@gmail.com  Wed Jan 27 14:32:25 2010
From: Roel Kluin <roel.kluin@gmail.com>
Date: Sun, 17 Jan 2010 16:50:10 +0100
Subject: Staging: serqt_usb2: Two branches the same in qt_set_termios()
To: Greg Kroah-Hartman <gregkh@suse.de>, devel@driverdev.osuosl.org, Andrew Morton <akpm@linux-foundation.org>, Bill Pemberton <wfp5p@virginia.edu>
Message-ID: <4B5331B2.8010604@gmail.com>

Regardless of the condition, the branches executed the same code

Signed-off-by: Roel Kluin <roel.kluin@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/serqt_usb2/serqt_usb2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/serqt_usb2/serqt_usb2.c
+++ b/drivers/staging/serqt_usb2/serqt_usb2.c
@@ -1277,7 +1277,7 @@ static void qt_set_termios(struct tty_st
 	if (cflag & CSTOPB)
 		new_LCR |= SERIAL_TWO_STOPB;
 	else
-		new_LCR |= SERIAL_TWO_STOPB;
+		new_LCR |= SERIAL_ONE_STOPB;
 
 	dbg("%s - 4\n", __func__);
 
